docs/pages/includes/helm-reference/zz_generated.teleport-kube-agent.mdx

```diff
@@ -1127,11 +1127,7 @@ For this reason, it is strongly discouraged to set a custom image when using
 automatic updates. Teleport Cloud uses automatic updates by default.
 </Admonition>
 
-Since version 13, hardened distroless images are used by default.  You can use
-the deprecated debian-based images by setting the value to
-`public.ecr.aws/gravitational/teleport`. Those images will be removed with
-teleport 15.
-
+Hardened distroless images are used by default.
```

Comment:
This sentence is hard to understand for people without Teleport nor distroless context. Can you add a link to https://goteleport.com/docs/installation/#interacting-with-distroless-images ?

Reply:
Changed in 3e57f58145a164a513397cda0eb23832aa503927. I added a descriptive sentence rather than a link so the reader could stay on the same page.